pkg/notifications/new_discussion_replied_meta_test.go

```diff
@@ -0,0 +1,77 @@
+package notifications
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+)
+
+func (suite *NotificationsSuite) TestActivityNewDiscussionRepliedCreate() {
```

Comment:
This is a good start, you probably could do without any tags in this test, the other activity type really deals with the tagged notification. I'd expect assertions that
1. The user that made the discussion gets a notification for the reply, and then assert the content matches as expected.